emojirades/commands/gamestate_commands/correct_guess_command.py

```diff
@@ -83,10 +83,31 @@ def execute(self):
                 "Holy bejesus Batman :bat::man:, they guessed it in one go! :clap:",
             )
 
+        # Build the score message
+        if score == 1000:
+            prefix = (
+                f"Ok <@{state['winner']}> just give up already, you've won the game"
+            )
```

Comment:
what's with the brackets here?

Reply:
Black decided to do this to put the string on a new line